weasyl/templates/etc/site_update.html

```diff
@@ -6,7 +6,7 @@ <h3>${update.title}</h3>
 
     <figure class="thumbnail thumbnail-left clear">
       <a class="avatar" href="/~${owner.login_name}" title="${owner.profile.username}">
-        <img src="${owner.avatar['display_url']}" alt="avatar of ${owner.profile.username}" />
+        <img src="${owner.avatar_display_url or resource_path('img/default-avatar.jpg')}" alt="avatar of ${owner.profile.username}" />
```

Comment:
More logic in templates :(
Will add looking into moving this logic back to code in the new templates